:memo: Update SSH guide (#2222)
platformsh/platformsh-docs: docs/src/development/ssh/ssh-keys.md

Include one `Host` entry for each Platform.sh region you want to connect to, such as `us-2` or `eu-4`.
(You can include other configuration as desired.)

## Troubleshooting

A given SSH key pair can only be linked to a single user account.

> Attempting to add the same SSH key to another account will result in the `SSH key can not be duplicated` error message.

Reviewer: ```suggestion
If you add an already used SSH key to another account, you see the error: `SSH key can not be duplicated`.
```

> Generate a new pair of SSH keys for the second user account you want to add.

Reviewer: ```suggestion

To resolve this error, generate separate pairs of SSH keys for each account.
```

You can use several SSH profiles with the `~/.ssh/config` file.

> More information about that file and its capabilities can be retrieved on your local machine with the `man ssh_config` command.

Reviewer: I don't know much about this, but it seems confusing to me. Do the profiles work instead of adding additional keys? In addition to? I'd consider either adding more context about their purpose with this error or else remove this part. I don't know close to enough to say which would be better. :shrug:
